gadget.yaml

```diff
@@ -7,7 +7,7 @@ volumes:
         role: system-seed
         filesystem: vfat
         type: 0C
-        size: 1200M
+        size: 3500M
```

Comment:
If you plan to do things like a remodel from older gadget to this one, using `min-size: 1200M` would keep partitions compatible and make that possible.

Reply:
This gadget is published as pi-desktop, which is new.  But in the future we may need to consider that if we ever need to grow beyond 3500M.

Reply:
It's entirely possible that we don't want a snap that's published under a different name in the source tree.  The current pi-desktop snap is built/published from another repo under the canonical org.  I'd like to hear what others think.  Keeping it here in a branch will ease merging fixes/improvements from non-desktop branches.